src/common/const.go

```diff
@@ -104,6 +104,7 @@ const (
 	HTTPAuthProxyTokenReviewEndpoint = "http_authproxy_tokenreview_endpoint"
 	HTTPAuthProxyVerifyCert          = "http_authproxy_verify_cert"
 	HTTPAuthProxySkipSearch          = "http_authproxy_skip_search"
+	HTTPAuthProxyCaseSensitive       = "http_authproxy_case_sensitive"
```

Comment:
The case sensitive might be used by other auth mod, for example, LDAP

Reply:
Is LDAP case-insensitive